pkg/mesh/helpers.go

```diff
@@ -0,0 +1,54 @@
+package mesh
+
+import (
+	"github.com/deislabs/smi-sdk-go/pkg/apis/metrics"
+	v1 "k8s.io/api/core/v1"
+)
+
+type contextKey string
+
+var (
+	IntervalKey = contextKey("metrics-interval")
```

Comment:
Why’d you move these out of `metrics`? They’re only used by the handler and helpers.

Reply:
opps, Right! moved them back!